src/styles/style.css

```
@@ -311,16 +291,19 @@ img {
 	}
 
 	& .cssdb-feature-polyfill-stars-minimal {
-		margin-inline-start: 3px;
 		overflow: hidden;
 
 		& img {
 			height: 20px;
-			transform: translateX(-58px);
+			margin-inline-start: -56px;
```

```suggestion
			margin-inline-start: -56px;
			min-width: 57px;
```

To make image lazy loading work.
Stars did not appear for me in latest Chrome.